truss/cli/cli.py

```diff
@@ -945,6 +947,22 @@ def get_job_logs(remote: Optional[str], project_id: str, job_id: str, tail: bool
     remote_provider: BasetenRemote = cast(
         BasetenRemote, RemoteFactory.create(remote=remote)
     )
+    if not project_id or not job_id:
```

Comment:
How would you feel about an experience that surfaces logs from the most recent by default (with a warning or something), and allows them to provide the IDs explicitly if needed?

Or like a `--choose` option as well that shows a table, then clicking enter grabs the logs for a specific job